core/domain/feedback_services.py

```diff
@@ -90,8 +90,10 @@ def create_thread(
         subject: str. The subject of this thread.
         text: str. The text of the feedback message. This may be ''.
     """
-    _create_models_for_thread_and_first_message(
+    thread_id = _create_models_for_thread_and_first_message(
         exploration_id, state_name, original_author_id, subject, text, False)
+    return feedback_models.FeedbackThreadModel.generate_full_thread_id(
```

Comment:
I thought we were getting rid of all the thread ID / full thread ID stuff?

Reply:
Yes @seanlip. I'll edit all these locations after the other PR goes in.

Reply:
Ah, ok. There should be a comment here then, like all the other places. Without a comment there is an implicit assumption that the code is final.

Reply:
Oh ok. My bad. Will comment above all locations I plan to edit after that PR goes in. Thanks!